### DISCUSSION TITLE
Ability to associate a piece of code with a tag

### DISCUSSION THREAD

**Discussion by radha1990kumari** [SilentVoid13/Templater#347] [Ideas & Feature Requests]
Example 
```js

{custom}<%*
let re = new RegExp('Lab-(\\d+)_.*');
var myString = tp.file.title;
var match = re.exec(myString); 
var an="0"
if(match!=null)
an=match[1]
tR=an
%>
```
So wherever in the template I use `{custom}` the output of the code gets replaced .
Let  me know if we can do this already.

**Comment by welpdx** [comment]
Hey,

As mentioned [here in your previous question](https://github.com/SilentVoid13/Templater/discussions/348#discussioncomment-1299610), the line: `tR=an` replaced the output with only `an`. Replace that line with `tR+=an`, and your problem is solved.

Hope that helps!

Welp

**Reply by radha1990kumari** [reply]
@welpdx , this is a separate  query

**Reply by welpdx** [reply]
Hi @radha1990kumari, I understand that this is a diff question. But I believe that same fix would apply to both of your questions/problems. 

Edit: clarification
